packages/create-workleap/index.ts

```diff
@@ -0,0 +1,109 @@
+import * as child_process from "child_process";
+import process from "process";
+
+import { Intro, Select, Note, Text, Outro } from "./prompts";
+import * as pkg from "./package.json";
+import { StartSpinner, StopSpinner } from "./spinner";
+
+const DEFAULT_OUTPUT_DIRECTORY = "./my-new-project";
+const NAME_PARAMETER_POSITION = 2; // TODO validate position of parameter once ask with `pnpm create`
+
+const FOUNDRY_CMD = "foundry";
+
+const AskForOutputDirectoryAsync = async (): Promise<string> => {
```

Comment:
I am not sure why you have to extract this code in a function when it's only used at a single place.

Why not declare it directly inline in the main function? Easier to read as I don't have to jump between the function and the main function.